test/infrastructure/docker/config/webhook/service.yaml

```diff
@@ -7,6 +6,5 @@ metadata:
 spec:
   ports:
     - port: 443
-      targetPort: 9443
-  selector:
-    control-plane: controller-manager
+      targetPort: webhook-server
```

Comment:
This is an inconsistency in CAPD vs other providers

Reply:
Can you provide some more context what the inconsistency is, why we have it and why we are dropping the selector now?

Reply:
This is mostly for consistency across al the providers in the CAPI codebase, eg.

Reply:
Ah got it, so we're consistent now after this change. Wasn't sure ;).